src/libsync/vfs/suffix/vfs_suffix.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) by Christian Kamm <mail@ckamm.de>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#include "vfs_suffix.h"
+
+#include <QFile>
+
+#include "syncfileitem.h"
+#include "filesystem.h"
+
+namespace OCC {
+
+class VfsSuffixPrivate
+{
+};
+
+VfsSuffix::VfsSuffix(QObject *parent)
+    : Vfs(parent)
+    , d_ptr(new VfsSuffixPrivate)
+{
+}
+
+VfsSuffix::~VfsSuffix()
+{
+}
+
+Vfs::Mode VfsSuffix::mode() const
+{
+    return WithSuffix;
+}
+
+QString VfsSuffix::fileSuffix() const
+{
+    return QStringLiteral(APPLICATION_DOTVIRTUALFILE_SUFFIX);
+}
+
+void VfsSuffix::registerFolder(const VfsSetupParams &)
+{
+}
+
+void VfsSuffix::start(const VfsSetupParams &)
+{
+}
+
+void VfsSuffix::stop()
+{
+}
+
+void VfsSuffix::unregisterFolder()
+{
+}
+
+bool VfsSuffix::isHydrating() const
+{
+    return false;
+}
+
+bool VfsSuffix::updateMetadata(const QString &filePath, time_t modtime, quint64, const QByteArray &, QString *)
+{
+    FileSystem::setModTime(filePath, modtime);
+    return true;
+}
+
+void VfsSuffix::createPlaceholder(const QString &syncFolder, const SyncFileItemPtr &item)
+{
+    // NOTE: Other places might depend on contents of placeholder files (like csync_update)
```

Comment:
csync_update lo longer exist.  maybe time to update the comment.

Reply:
Can be removed now since the only place that depends on it is a few lines down, `isDehydratedPlaceholder`.